src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoNotNullable.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is the showing up as changed - is this a newline issue? Same below in the other file.

Reply:
``` -<EF><BB><BF>// Licensed to the .NET Foundation under one or more agreements.``` 
There was some extra characters,. I have seen them sometimes before as well when we copy these lines. so I just removed these extra characters.